### PR TITLE
🍼 Populate flow template dependencies

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/flows/tests.py [hunk 1]
```diff
@@ -5956,3 +5956,46 @@ def test_trim_revisions(self):
         trim_flow_revisions()
         self.assertEqual(2, FlowRevision.objects.filter(flow=clinic).count())
         self.assertEqual(31, FlowRevision.objects.filter(flow=color).count())
+
+
+class PopulateTemplateDepsTest(MigrationTest):
```
Comment (rowanseymour) [inline temba/flows/tests.py]: sometimes I wonder how on earth we did data migrations without unit testing them.. I rarely get it 100% before testing it

File: temba/flows/migrations/0225_populate_template_deps.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Generated by Django 2.2.4 on 2020-02-07 15:26
+
+from django.db import migrations, transaction
+
+BATCH_SIZE = 5000
+
+
+def populate_template_deps(apps, schema_editor):  # pragma: no cover
```
Comment (nicpottier) [inline temba/flows/migrations/0225_populate_template_deps.py]: why the no cover if you are testing?

Comment (rowanseymour) [reply temba/flows/migrations/0225_populate_template_deps.py]: cuz I copied and pasted this from another migration 🤦‍♂

**Comment by rowanseymour** [comment]
@nicpottier not following.. this is creating db dependencies for templates.. but we need the dependencies in metadata to be in the right format for the editor to consume. Bear in mind that missing dependencies will be listed in the metadata but not in the db for obvious reasons.

**Comment by nicpottier** [comment]
Ya, just saying since this is a full recalc / update of dependencies, this migration could actually cover the other one too by just setting the dependency metadata. Obviously that one needed to happen first though!

**Comment by rowanseymour** [comment]
Ah yeah could have done some migration batching if I'd planned ahead!

**Pull request merged**